src/app/components/PlayersBoard.ts

```diff
@@ -64,12 +69,50 @@ export class PlayersBoard {
       playersDataPanel,
       playerName,
       playerAvatar,
+      playerTimer,
     );
     Render.childrenInjector(
       playersBoardContainer,
       playersDataPanel,
       herdView,
     );
+    /*
+    this.view.stopTimer();
+    setTimeout(() => this.hideTimer(), 10);
+  }
+
+  private hideTimer(): void {
+    (document.querySelector(
+      '#time-left',
+    ) as HTMLElement).style.display = 'none';
+  }
```

Comment:
this will probably have to target a specific timer in a specific players container.